packages/cojson/src/coValues/coMap.ts

```diff
@@ -49,6 +49,9 @@ export class RawCoMapView<
   cachedOps?: {
     [Key in keyof Shape & string]?: MapOp<Key, Shape[Key]>[];
   };
+  /** @internal */
+  validSortedTransactions?: DecryptedTransaction[];
```

Comment:
This can also be cached, guess that's also where most of the overhead comes from.